packages/devtools/lib/src/logging/logging.dart

```diff
@@ -571,7 +571,7 @@ class LogWhenColumn extends Column<LogData> {
   bool get supportsSorting => false;
 
   @override
-  dynamic getValue(LogData item) => item.timestamp;
```

Comment:
was this refactor intentional of did the dart refactor tool get carried away and rename this even though you didn't mean to?

Reply:
This was intentional. We were calling objects both `item` and `dataObject` so I changed them all to `dataObject` to be consistent.